README.md

```diff
@@ -108,6 +108,8 @@ $ make
 
 ### Build for DRM backend
 
+This binary will run by setting the DRM device file name(/dev/dri/card xx) to the environment variable `DRM_DEVICE_FILENAME`.
```

Comment:
The location isn't properly. And please add an example. The text is not intuitive.

Reply:
I moved the description and add an example.

Reply:
I fixed it according to other comments.

Reply:
What does it mean?

Reply:
Sorry, I misunderstood moving this description to `NOTE` section.  
Now, this description is removed and I added the same mean description to `NOTE` section.  Is this fix OK?

Reply:
Sure.